tests/Assistants/VectorStoreTests.cs

```diff
@@ -73,7 +73,7 @@ public void CanCreateGetAndDeleteVectorStores()
         Assert.That(deleted, Is.True);
         _vectorStoresToDelete.RemoveAt(_vectorStoresToDelete.Count - 1);
 
-        vectorStore = client.CreateVectorStore(new()
+        vectorStore = client.CreateVectorStore(new VectorStoreCreationOptions()
```

Comment:
Is there any reasonable approach for us to avoid the overload collisions that make implicit `new()` non-viable? It's a small thing but it's unfortunate if we miss an opportunity to make instantiation of these trivial options containers simpler. Would it e.g. be problematic for us to have three declarations:
- Foo(FooOptions, CancellationToken)
- Foo(FooOptions) => Foo(FooOptions, default(CancellationToken))
- Foo(BinaryContent, RequestOptions)

Reply:
Yeah, we could do this. But ultimately, it means we should not use optional parameters in public APIs. Even that does not solve the evolution problem completely, but it does mitigate it.